libs/designer/src/lib/ui/panel/recommendation/browseView.tsx

```diff
@@ -75,6 +75,7 @@ export const BrowseView = (props: BrowseViewProps) => {
   const allApiIdsWithActions = useAllApiIdsWithActions();
   const allApiIdsWithTriggers = useAllApiIdsWithTriggers();
 
+  // TODO: This callback is expensive, consider optimizing it
```

Comment:
This is not specific to this PR, just something I noticed, I will be investigating it more later